tox.ini

```diff
@@ -18,23 +18,23 @@ setenv =
 passenv = *
 
 commands =
-    /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
+;    /usr/bin/find "{toxinidir}" -name '*.pyc' -delete
```

Comment:
Is this on purpose?

Reply:
Yeah, the second run failed, see https://github.com/matrix-org/synapse-s3-storage-provider/actions/runs/3876182424/jobs/6609721762#step:5:24

My reading of this: tox complained because we were invoking an external binary that tox wasn't responsible for installing.

Reply:
I could just delete the line altogether, I suppose?

Reply:
Oh, weirdly I think if we don't do `usedevelop` than this won't matter and we can just remove this line?

Reply:
I'm doubtful that this has any effect at all, tbh. IIRC pyc files store a hash of the underlying module file, so that the interpreter can detect when a pyc file is out-of-date. I'm not sure what we get by deleting old bytecode caches.

But let's nuke usedevelop and remove this too.